src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Machine - buldozer started working");
```

Comment:
The message should be more informative and indicate the specific type of `Machine` that is working. Consider changing the message to something like 'Bulldozer started its work.' to clearly reflect the type of machine.